tests/test_density_RDF.py

```diff
@@ -79,34 +79,36 @@ def test_random_point(self):
         tolerance = 0.1
         box_size = r_max * 3.1
 
-        for i, r_min in enumerate([0, 0.05, 0.1, 1.0, 3.0]):
+        for r_min in (0, 0.1, 3.0):
             box, points = freud.data.make_random_system(box_size, num_points)
-            test_set = util.make_raw_query_nlist_test_set(
-                box, points, points, "ball", r_max, 0, True
+            # This test is slow, and since it's a validation of the underlying
+            # algorithm and not the API we don't need to test all possible
+            # inputs, so we only test the fastest one (AABBQuery).
+            nq = freud.locality.AABBQuery(box, points)
+            neighbors = {"mode": "ball", "r_max": r_max, "exclude_ii": True}
+
+            rdf = freud.density.RDF(bins, r_max, r_min)
+
+            if r_min != 3.0:
+                rdf.compute(nq, neighbors=neighbors, reset=False)
+            else:
+                rdf.compute(nq, neighbors=neighbors)
+
```

Comment:
You removed the assertion which compares `rdf.box`. Is there a specific reason behind that?

Reply:
```suggestion

            assert rdf.box == box
```